package.json

```diff
@@ -98,32 +98,23 @@
     "node": ">=12.18.4"
   },
   "devDependencies": {
-    "@babel/compat-data": "^7.26.8",
-    "@mongodb-js/monorepo-tools": "^1.1.10",
+    "@mongodb-js/monorepo-tools": "^1.1.16",
     "@pkgjs/nv": "^0.2.2",
     "@types/chai": "^4.2.5",
     "@types/mocha": "^5.2.7",
     "@types/node": "^22.15.30",
-    "@types/rimraf": "^3.0.0",
     "@types/semver": "^7.3.4",
     "@types/sinon-chai": "^3.2.4",
-    "@types/which": "^1.3.2",
     "chai": "^4.2.0",
     "cross-env": "^6.0.3",
-    "depcheck": "^1.4.7",
-    "duplexpair": "^1.0.2",
-    "find-up": "^5.0.0",
     "glob": "^10.3.12",
     "husky": "^9.0.11",
+    "knip": "^5.69.1",
     "mocha": "^10.2.0",
-    "mongodb": "^6.19.0",
     "mongodb-runner": "^6.0.0",
-    "node-gyp": "^9.0.0 || ^10.2.0",
     "nyc": "^15.1.0",
-    "pkg-up": "^3.1.0",
     "rimraf": "^3.0.2",
-    "semver": "^7.6.3",
-    "sinon": "^7.5.0",
+    "sinon": "^19.0.4",
```

Comment:
btw, we should probably remove all or most packages here over time, this is a great step in that direction